application/org.openjdk.jmc.console.agent/src/main/java/org/openjdk/jmc/console/agent/manager/wizards/PresetManagerPage.java

```diff
@@ -206,13 +206,15 @@ protected void onImportFilesButtonSelected(IStructuredSelection selection) {
 				}
 
 				tableInspector.getViewer().refresh();
+				tableInspector.setFocus();
```

Comment:
That was my original suggestion, but since there's `setFocus` (a single statement), but at the same time it's just a suggestion that does not bring as much value since the goal of `setFocus` is somewhat straightforward.

Reply:
I have added setFocus to Import Button as well as Export Button. I have extracted that to a method now.